### PR TITLE
Panorama - added vsys argument to the panorama command

### PR DESCRIPTION
**Pull request by reutshal**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/44963

## Description
Added vsys argument to the `panorama` command

@bakatzir Please also take a look at the argument description and the RN. Thanks!

### CODE REVIEW

File: Packs/PAN-OS/Integrations/Panorama/Panorama.yml [hunk 1]
```diff
@@ -134,6 +134,8 @@ script:
       name: job-id
     - description: Query string.
       name: query
+    - description: The name of the virtual system to be configured. If no vsys is mentioned, this command will not pass the vsys parameter to the http request.
```
Comment (bakatzir) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.yml:137]:
```suggestion
    - description: The name of the virtual system to be configured. If no vsys is mentioned, this command will not  use the vsys parameter.
```

File: Packs/PAN-OS/Integrations/Panorama/README.md [hunk 2]
```diff
@@ -213,6 +213,7 @@ Run any command supported in the API.
 | target | Target number of the firewall. Use only on a Panorama instance. | Optional | 
 | job-id | Job ID. | Optional | 
 | query | Query string. | Optional | 
+| vsys | The name of the virtual system to be configured. If no vsys is mentioned, this command will not pass the vsys parameter to the http request. | Optional |
```
Comment (bakatzir) [inline Packs/PAN-OS/Integrations/Panorama/README.md:216]:
```suggestion
| vsys | he name of the virtual system to be configured. If no vsys is mentioned, this command will not  use the vsys parameter. | Optional | 
```

File: Packs/PAN-OS/ReleaseNotes/1_8_1.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Palo Alto Networks PAN-OS
+- Added the **vsys** argument to the ***panorama*** command which will be passed as a parameter to the http request.
```
Comment (bakatzir) [inline Packs/PAN-OS/ReleaseNotes/1_8_1.md:4]:
```suggestion
- Added the **vsys** argument to the ***panorama*** command.
```

**Review by bakatzir — APPROVED** [review]

**Review by yaakovi — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@reutshal I agree with Bar's comment and made the change in the files.  Small change in the RN.

**Pull request merged**